README.md

```diff
@@ -11,69 +11,54 @@ See [what's deployed](https://whatsdeployed.io/s-J18)
+and follow the following steps.
 
 1. Fork this repository & clone it to your local machine.
-
-1. Download base Kitsune docker images:
    ```
-   docker pull mozmeao/kitsune:base-latest
-   docker pull mozmeao/kitsune:base-dev-latest
+   git clone https://github.com/mozilla/kitsune.git
    ```
 
-1. Build Kitsune docker images. (Only needed on initial build or when packages change)
+2. Pull base Kitsune Docker images, create your database, and install node and bower packages.
    ```
-   docker-compose -f docker-compose.yml -f docker/composefiles/build.yml build base
-   docker-compose -f docker-compose.yml -f docker/composefiles/build.yml build dev
+   make init
    ```
 
-1. Copy .env-dist to .env
+3. Run Kitsune.
    ```
-   cp .env-dist .env
+   make run
    ```
 
-1. Create your database
-   ```
-   docker-compose -f docker-compose.yml -f docker/composefiles/dev.yml run web ./manage.py migrate
-   ```
-
-1. Install node and bower packages
-   ```
-   docker-compose -f docker-compose.yml -f docker/composefiles/dev.yml run web yarn
-   docker-compose -f docker-compose.yml -f docker/composefiles/dev.yml run web ./node_modules/.bin/bower install --allow-root
-   ```
-
-1. (Optional) Enable the admin control panel
-   ```
-   echo "ENABLE_ADMIN=True" >> .env
-   ```
+The running instance will be located at http://0.0.0.0:8000/ unless you specified otherwise, and the administrative control panel will be at http://0.0.0.0:8000/admin.
 
-1. Run Kitsune
-   ```
-   docker-compose -f docker-compose.yml -f docker/composefiles/dev.yml up web
-   ```
+After that you can do some additional optional steps if you want to for the admin:
 
-   The running instance will be located at http://0.0.0.0:8000/ unless you specified otherwise, and the administrative control panel will be at http://0.0.0.0:8000/admin.
+* Enable the admin control panel
+  ```
+  echo "ENABLE_ADMIN=True" >> .env
+  ```
 
-1. (Optional) Create a superuser
-   ```
-   docker-compose -f docker-compose.yml -f docker/composefiles/dev.yml exec web ./manage.py createsuperuser
-   ```
+* Create a superuser
+  ```
+  docker-compose exec web ./manage.py createsuperuser
+  ```
 
-1. (Optional) Create some data
-   ```
-   docker-compose -f docker-compose.yml -f docker/composefiles/dev.yml exec web ./manage.py generatedata
-   ```
+* Create some data
+  ```
+  docker-compose exec web ./manage.py generatedata
+  ```
 
-1. (Optional) Update product details
-   ```
-   docker-compose -f docker-compose.yml -f docker/composefiles/dev.yml exec web ./manage.py update_product_details
-   ```
+* Update product details
+  ```
+  docker-compose exec web ./manage.py update_product_details
+  ```
 
-1. (Optional) Get search working
+* Get search working
 
    First, make sure you have run the "Create some data" step above.
 
-   1. Enter the web container: `docker exec -it kitsune_web_1 /bin/bash`
+   1. Enter the web container: `docker-compose exec web bash`
```

Comment:
is there any reason that this isn't done automatically during `make run`?

Reply:
I think the idea is that many (most) people won't need this and so it's better to avoid the time it takes to do these steps at first. I could be convinced otherwise though. Also these require some user input, so I don't mind them being separate, but could be better incorporated I'm sure.